CMakeLists.txt

```diff
@@ -333,6 +333,15 @@ if(VITA)
 
 endif()
 
+if(PS4)
+  add_custom_command(
+    TARGET ${BIN_TARGET} POST_BUILD
+    COMMAND ${CMAKE_COMMAND} -E copy ${DEVILUTIONX_MPQ} "${PROJECT_SOURCE_DIR}/Packaging/ps4/")
+  add_self(${BIN_TARGET})
+  add_pkg(${BIN_TARGET} "${PROJECT_SOURCE_DIR}/Packaging/ps4"

```

Comment:
Shoudn't this say 1.4.0 instead of 1.00?

Reply:
Ideally, I guess so. I just copied what was done for the vita, which is similar to the ps4 in many ways. Blindly using ${PROJECT_VERSION} yeild errors, e.g., "PKG Content ID must be 36 characters long.". I'll see if I can address this in the sdk.

Reply:
Made a couple of changes to pacbrew sdk to make it a bit more robust against incomparable version strings, so using version generated from git hash etc will not break the build system now